src/Aspire.Dashboard/Otlp/Model/OtlpHelpers.cs

```diff
@@ -455,15 +455,30 @@ public static bool TryAddScope(Dictionary<string, OtlpScope> scopes, Instrumenta
             var name = scope?.Name ?? string.Empty;
             ref var scopeRef = ref CollectionsMarshal.GetValueRefOrAddDefault(scopes, name, out _);
             // Adds to dictionary if not present.
-            scopeRef ??= (scope != null) ? new OtlpScope(scope.Name, scope.Version, scope.Attributes.ToKeyValuePairs(context)) : OtlpScope.Empty;
+            if (scopeRef == null)
+            {
+                scopeRef = (scope != null)
+                    ? new OtlpScope(scope.Name, scope.Version, scope.Attributes.ToKeyValuePairs(context))
+                    : OtlpScope.Empty;
+
+                context.Logger.LogTrace("Added scope '{ScopeName}' to {TelemetryType}.", scopeRef.Name, telemetryType);
+            }
+
             s = scopeRef;
             return true;
         }
         catch (Exception ex)
         {
-            context.Logger.LogInformation(ex, "Error adding scope.");
+            context.Logger.LogInformation(ex, "Error adding scope to {TelemetryType}.", telemetryType);
```

Comment:
Why is this info?

Reply:
I don't remember why info was used.

I think it might be because info isn't printed to the app host console (don't want continious errors from importing telemetry) but it's more important than other telemetry debug and trace logs.